qemu/tests/balloon_boot_in_pause.py

```diff
@@ -0,0 +1,221 @@
+import logging
+import random
+
+from avocado.core import exceptions
+
+from virttest import error_context
+from virttest import utils_misc
+from qemu.tests.balloon_check import BallooningTest
+
+
+class BallooningTestPause(BallooningTest):
+
+    """
+    Basic functions of memory ballooning test for guest booted
+    in paused status
+    """
+
+    def __init__(self, test, params, env):
+        super(BallooningTest, self).__init__(test, params, env)
+
+        self.vm = env.get_vm(params["main_vm"])
+        self.ori_mem = self.get_vm_mem(self.vm)
+        if self.get_ballooned_memory() != self.ori_mem:
+            self.balloon_memory(self.ori_mem)
+        self.old_mmem = self.ori_mem
```

Comment:
@aliang123 We can not use "self.ori_mem" to check the ballooned memory SINCE the original guest memory could not obtained in the case scenario